engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -167,13 +168,20 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
     /*
      Commands from clients
      */
-    case Event(RetrieveNewWorkflows, stateData) =>
+    case Event(RetrieveNewWorkflows, _) =>
       /*
         Cap the total number of workflows in flight, but also make sure we don't pull too many in at once.
         Determine the number of available workflow slots and request the smaller of that number and maxWorkflowsToLaunch.
        */
+      params.jobExecutionTokenDispenserActor ! FetchLimitedGroups
+      stay()
+    case Event(ReplyLimitedGroups(groups), stateData) =>
+      if (groups.nonEmpty)
+        log.info(s"Excluding groups from workflow launch: ${groups.mkString(", ")}")
+      else
+        log.debug("No groups excluded from workflow launch.")
```

Comment:
Is there a way this could be recorded via metrics rather than debug messages? (AFAIK we don't emit debug right now?)

Reply:
I could hack something together but it seems like a better move to wait for [this ticket](https://broadworkbench.atlassian.net/browse/BW-1121) that does a comprehensive metrication.

Reply:
`debug` vs. `info` above is so that the "nothing is happening" message doesn't spam the prod logs for the majority of the time that it is the current message.

Reply:
+1 to waiting for that ticket.